### PR TITLE
Adapt notebook to show rebinning

### PR DESCRIPTION
**Pull request by AtreyeeS**
This adapts the Variability tutorial after #5007 
Thanks to @cgalelli for figuring out it was an issue with the time axis scaling!

### CODE REVIEW

**Review by cgalelli — APPROVED** [review]
Thanks for the quick fix @AtreyeeS !

File: examples/tutorials/analysis-time/Variability_estimation.py [hunk 1]
```diff
@@ -187,39 +189,29 @@
...
+######################################################################
+# We can rebin the lightcurve to compute the one expected with bayesian edges
+# First, we adjust the first and last bins of the bayesian_edges to coincide
+# with the original light curve start and end points.
+
+# create a new axis
+axis_original = lc_1d.geom.axes["time"]
+bayesian_edges[0] = axis_original.time_edges[0].value
+bayesian_edges[-1] = axis_original.time_edges[-1].value
+edges = Time(bayesian_edges, format="mjd", scale=axis_original.reference_time.scale)
```
Comment (registerrier) [inline examples/tutorials/analysis-time/Variability_estimation.py:208]: Isn't it possible to pass `bayesian_blocks` a `Time` object directly?

**Review by registerrier — APPROVED** [review]
Thanks @AtreyeeS and @cgalelli . This looks good!

File: examples/tutorials/analysis-time/Variability_estimation.py (hunk 1 above)
Comment (AtreyeeS) [reply examples/tutorials/analysis-time/Variability_estimation.py:208]: No! Maybe it is worth filing an issue in astropy.

**Pull request merged**